custom_components/tuya_local/devices/airrobo_p20_vacuum.yaml

```diff
@@ -75,16 +80,17 @@ entities:
           - dps_val: gentle
             value: Silent
           - dps_val: normal
-            value: Normal
+            value: Standard
           - dps_val: strong
-            value: Strong
+            value: High
       - id: 18
         type: bitfield
         name: error
-        optional: true
+        hidden: true

```

Comment:
Removing the optional from dp 18 will affect device detection, I think it is there for a reason. Hiding it is OK, since there is a separate problem sensor, and it only needs to be here to report error as the entity state.